efficacy/app.R

```diff
@@ -11,52 +11,65 @@ options(shiny.useragg = FALSE)
 
 nest_logo <- "https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/nest.png"
 
-# code>
-## Generate Data
-ADSL <- synthetic_cdisc_data("latest")$adsl
-adsl_labels <- teal.data::col_labels(ADSL, fill = FALSE)
+## Data reproducible code ----
+data <- teal_data()
+data <- within(data, {
```

Comment:
I've included library calls in all `teal_data` now. There will be duplicated library calls
- library calls included in pre-processing manually
- library calls added by teal to the SRC returned code

I propose to keep duplicated for a while and remove them from `get_rcode_header` later. It will also be a hot topic to discuss as the issue isn't really easy to solve.

Reply:
Right, I forgot about the header. Fair enough, separate issue it is.